inspirehep/records/serializers/latex.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+import os
+
+import jinja2
+import pkg_resources
+from invenio_records_rest.serializers.json import MarshmallowMixin, PreprocessorMixin
+from invenio_records_rest.serializers.response import (
+    record_responsify,
+    search_responsify,
+)
+
+from ..marshmallow.literature.latex import LatexSchema
+
+
+class LatexSerializer(MarshmallowMixin, PreprocessorMixin):
+    """Latex serializer for records."""
+
+    def __init__(self, format, **kwargs):
+        """Initialize record."""
+        self.format = format
+        super(LatexSerializer, self).__init__(**kwargs)
+
+    def serialize(self, pid, record, links_factory=None, **kwargs):
+        """Serialize a single record and persistent identifier.
+
+        :param pid: Persistent identifier instance.
+        :param record: Record instance.
+        :param links_factory: Factory function for record links.
+        """
+        data = self.transform_record(pid, record, links_factory, **kwargs)
+
+        return self.latex_template().render(data=data, format=self.format)
+
+    def preprocess_record(self, pid, record, links_factory=None, **kwargs):
+        """Prepare a record and persistent identifier for serialization."""
+        return record
+
+    def latex_template(self):
+        latex_jinja_env = jinja2.Environment(
+            variable_start_string="\VAR{",
+            variable_end_string="}",
+            loader=jinja2.FileSystemLoader(os.path.abspath("/")),
+        )
+        template_path = pkg_resources.resource_filename(
+            "inspirehep", "records/serializers/templates/latex_template.tex"
+        )
+
+        template = latex_jinja_env.get_template(template_path)
+
+        return template
+
+    def serialize_search(
+        self, pid_fetcher, search_result, links=None, item_links_factory=None
```

Comment:
Some parameters here are not used as well. In case you want to keep the same signature, you might want to have the same as https://github.com/inspirehep/inspirehep/pull/19/files#diff-91b17d92672862801bb8d53a5e34465aR25

Reply:
ditto https://github.com/inveniosoftware/invenio-records-rest/blob/master/invenio_records_rest/serializers/json.py#L36-L47